tests/test_elasticsearch_walker.py

```diff
@@ -47,6 +47,17 @@ def test_empty_query(self):
             "match_all": {}
         })
 
+    # Author queries
+    def test_doublequoted_author_search(self):
+        from invenio_search.api import Query
+        query_1 = Query("author: \"Jeffrey Goldstone\"").search().recids
```

Comment:
Normally there shouldn't be a space after the `:`. Have you checked it also works in that case?